ert3/engine/_run.py

```diff
@@ -79,19 +81,21 @@ def _transmitter_map_resources(
     futures: List[TransmitterCoroutine] = []
     for input_ in inputs:
         file_path = workspace.get_resources_dir() / input_.source_location
-        collection = ert.data.load_collection_from_file(
+
+        collection_awaitable = ert.data.load_collection_from_file(
```

Comment:
The return value of `load_collection_from_file` is indeed an awaitable, so therefore it was encoded in the variable name. It cannot be awaited in this source line since it is in a sync function, therefore we pass it as an awaitable to `transmit_record_collection`.

Reply:
Async black magic! 🪄